src/DocumentationParser.cpp

```diff
@@ -459,7 +459,7 @@ std::optional<std::string> WorkspaceFolder::getDocumentationForType(const Luau::
     {
         return printMoonwaveDocumentation(getComments(ftv->definition->definitionModuleName.value(), ftv->definition->definitionLocation));
     }
-    else if (auto ttv = Luau::get<Luau::TableType>(followedTy); ttv && !ttv->definitionModuleName.empty())
```

Comment:
Why did we remove this? won't this break the next line where we reference `ttv->definitionModuleName`?

Reply:
looks accidental